_posts/2021-08-11-solidity-0.8.7-release-announcement.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.7 Release Announcement'
+date: '2021-08-11'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.7](https://github.com/ethereum/solidity/releases/tag/v0.8.7) introduces support for
+the [London Hardfork](https://blog.ethereum.org/2021/07/15/london-mainnet-announcement/), includes
```

Comment:
The blog post uses the term "London upgrade" - not sure if that is important for us.

Reply:
Yeah, was wondering if the word "hardfork" should be avoided.

Reply:
fixed.